core/src/main/resources/error/error-classes.json

```diff
@@ -14,13 +18,21 @@
     "message" : [ "Found duplicate keys '%s'" ],
     "sqlState" : "23000"
   },
+  "EXCEED_ARRAY_SIZE_WHEN_ZIP_MAP" : {
+    "message" : [ "Unsuccessful try to zip maps with %s unique keys due to exceeding the array size limit %s." ],
+    "sqlState" : "22026"
```

Comment:
We can leave this blank for now; I don't see an obvious SQLSTATE for exceeding the array size limit (22026 is only for string data).